ci/build_common.sh

```diff
@@ -136,7 +140,7 @@ function configure_preset()
 
     pushd .. > /dev/null
 
-    cmake --preset=$PRESET --log-level=VERBOSE $GLOBAL_CMAKE_OPTIONS $CMAKE_OPTIONS
+    cmake --preset=$PRESET --log-level=VERBOSE $GLOBAL_CMAKE_OPTIONS "$ADDITIONAL_CMAKE_OPTIONS" $CMAKE_OPTIONS
```

Comment:
I would just append those to `GLOBAL_CMAKE_OPTIONS` instead of adding a new variable. That variable is meant for just this sort of thing.

Reply:
I tried that among the dozens of ways I tried to do this. The problem was any attempt I made to append the `ADDITIONAL_CMAKE_OPTIONS` would result in escaping the space(s) in the string.